app/components/sidebars/settings/settings_sidebar_base.js

```diff
@@ -269,6 +272,19 @@ export default class SettingsSidebarBase extends PureComponent {
             </View>
         );
 
+        const customStatusExpiryTime = isStatusSet && customStatus?.duration !== CustomStatusDuration.DONT_CLEAR ?
```

Comment:
does the expiry feature requires a specific server version?

Reply:
I think the changes we made in the server regarding the expiry feature will be in the 5.36 release. If that's true, then the expiry feature doesn't require a specific server version. The check we added for the custom status feature is enough for this feature.